src/kapil/Switch/BankSwitch.java

```diff
@@ -0,0 +1,165 @@
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+a) Person can debit or credit amount from respective bank.
+Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+c) A Method to show total credit, debit operations
+output : Total credit operation - 7 times
+Total debit operation - 5 times
+
+d) A Method to show individual bank credit operations.
+output : ICICI bank Credit Operation - 2
+HDFC bank Credit Operation - 2
+SBI bank Credit Operation - 1
+BOB bank Credit Operation - 4
+
+e) A Method to show individual bank debit operations.
+output : ICICI bank Debit Operation - 2
+HDFC bank Debit Operation - 2
+SBI
+
+*/
+package kapil.Switch;
+
+public class BankSwitch {
+       int iciciBalance = 40000;
+       int bobBalance = 30000;
+       int sbiBalance = 20000;
+       int hdfcBalance = 10000;
+       
+   int iciciDebitCount = 0, iciciCreditCount = 0,bobDebitCount = 0, bobCreditCount = 0, sbiDebitCount = 0, sbiCreditCount = 0, hdfcDebitCount=0, hdfcCreditCount=0;    
+   int totalDebitCount, totalCreditCount;
+   
+   
+   void operation1(String bank, String op, int amt) {
+	   switch(op) {
+	     case"Debit" :
+		   switch (bank) {
+		   case "icici" :
+	    	   if(iciciBalance>= amt) {
+	    		   iciciBalance = iciciBalance - amt;
+	          	   iciciDebitCount = iciciDebitCount + 1;	
+				   totalDebitCount = totalDebitCount + 1;
+				   System.out.println("icici debit amount is "+iciciBalance);
+				   }
+		
+	    	   else
+	    		   System.out.println("Balance is not sufficient ");
+	    	   break;
+		   case "bob" :
+	    	   if(bobBalance>= amt) {
+	    		   bobBalance = bobBalance - amt;
+	    	   bobDebitCount = bobDebitCount + 1;	
+			   totalDebitCount = totalDebitCount + 1;  
+	    	   System.out.println("bob debit amount is "+bobBalance);} 
+	    	   else
+	    		   System.out.println("Balance is not sufficient ");
+	    	   break;
+		   case "sbi" :
+	    	   if(sbiBalance>= amt) {
+	    		   sbiBalance = sbiBalance - amt;
+	    	   sbiDebitCount = sbiDebitCount + 1;	
+			   totalDebitCount = totalDebitCount + 1;
+	    	   System.out.println("sbi debit amount is "+sbiBalance);}
+	    	   else
+	    		   System.out.println("Balance is not sufficient ");
+	    	   break;
+		   case "hdfc" :
+	    	   if(hdfcBalance>= amt) {
+	    		   hdfcBalance = hdfcBalance - amt;
+	    	   hdfcDebitCount = hdfcDebitCount + 1;	
+			   totalDebitCount = totalDebitCount + 1;
+			   System.out.println("hdfc debit amount is "+hdfcBalance);
+		   }else
+	    		   System.out.println("Balance is not sufficient ");
+	    	   break;}
+		   break;
+	   }   	    	   
+		   
+		  
+	   } 
+	   void operation2(String bank, String op, int amount) {
+	   switch(op) {
+	     case"Credit" :
+		   switch (bank) {
+		   case "icici": 
+		      if(amount <= iciciBalance ) {
```

Comment:
What is use of this condition here?
Think on this, this means if credits amount is less then or equal to current balance then only you can credits amount.
think twice before writing code or after writing code think twice how your code behaves in different conditions.